fus.c

```diff
@@ -989,6 +989,57 @@ mask_non_default_module_pkgs (Pool *pool)
   return selection;
 }
 
+/*
+ * Mask bare rpms if any of the default modules provides them (even if older)
+ */
+static Queue
+mask_bare_rpms (Pool *pool, Map *modular_pkgs)
```

Comment:
Maybe I should have named it `mask_solvable_bare_rpms` because it only acts on RPMs which are in the list of solvables to be resolved.
We still need a `mask_bare_rpms` during resolution for masking RPMs that are dragged in as dependencies.

Reply:
I agree, the longer name is slightly better IMO.